yt/yt/ytlib/yql_client/public.h

```diff
@@ -16,6 +16,7 @@ DEFINE_ENUM(EExecuteMode,
 
 YT_DEFINE_ERROR_ENUM(
     ((Throttled)            (40100))
+    ((YqlAgentBanned)       (40101))
```

Comment:
What http/rpc proxy does when it is banned? Does it have a special error code for it?
If yes, then it would be nice to have one for QT too

Reply:
Yes it has its own error code, but we don't need it in qt. Maybe in YQL Agent it is redundant too. 

I have added an override for the IsUp function, which tells clients that if it returns false, this specific instance is unavailable. As a result, the client will stop seeing it and will stop sending requests to it.

I added this error to yqla because after the ban flag has been set to true, we have a short amount of time before the next isUp function call, during which we still accept new queries. These queries are too long, and we may want to stop accepting them as quickly as possible. In qt, we don't want this at all.